web/skins/classic/views/js/event.js

```diff
@@ -13,6 +13,7 @@ function changeScale()
     var newHeight = ( baseHeight * scale ) / SCALE_BASE;
 
     streamScale( scale );
+    Cookie.write( 'zmEventScale'+eventData.monitorId, scale, { duration: 10*365 } );
```

Comment:
I think `monitorId` needs to be `MonitorId` per the below change in `eventData`
Cookie is being created as `zmEventScaleundefined`